From 0ab7aca46ee68a1f13fe906faa12d1bfd8a74b15 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Thu, 22 Sep 2016 13:32:00 -0500
Subject: [PATCH 03/35] Revert "drm/tilcdc: Remove unnecessary
 tilcdc_crtc_disable() from tilcdc_unload()"

This reverts commit 299676e72141eeefe92df628c0410ee08724f4cb.
---
 drivers/gpu/drm/tilcdc/tilcdc_drv.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/gpu/drm/tilcdc/tilcdc_drv.c b/drivers/gpu/drm/tilcdc/tilcdc_drv.c
index 01183f4..a2f2474 100644
--- a/drivers/gpu/drm/tilcdc/tilcdc_drv.c
+++ b/drivers/gpu/drm/tilcdc/tilcdc_drv.c
@@ -200,6 +200,8 @@ static int tilcdc_unload(struct drm_device *dev)
 {
 	struct tilcdc_drm_private *priv = dev->dev_private;
 
+	tilcdc_crtc_disable(priv->crtc);
+
 	tilcdc_remove_external_encoders(dev);
 
 	drm_fbdev_cma_fini(priv->fbdev);
-- 
2.9.3

